debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -252,4 +279,14 @@ private static int validateCollectionBlacklist(Configuration config, Field field
         }
         return 0;
     }
+    
+    private static int validateDatabaseBlacklist(Configuration config, Field field, ValidationOutput problems) {
```

Comment:
So it turns out, combining includes and exludes aren't supported in the MySQL connector atm. either. Apparently it was meant to be supported at some point (there's some code in place for merging them), but then this validation was added (which you added to MongoDB, too) which prevents giving both options. So I'd suggest we do this for now (i.e. your PR is good as is), and then, as a follow-up task, we allow merging includes + excludes across all the connectors. WDYT?